src/Attribute/AdminDashboard.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace EasyCorp\Bundle\EasyAdminBundle\Attribute;
+
+/**
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class AdminDashboard
+{
+    public function __construct(
+        /** @var array<string, array{routeName: string, path: string}>|null */
```

Comment:
You could use nested attributes here for more type safety, if you think it makes sense

```suggestion
        /** @var array<string, AdminAction>|null */
```